openstack/nova/files/compute.nova.conf

```diff
@@ -117,18 +112,24 @@ user_domain_id = default
 auth_strategy = keystone
 admin_tenant_name = service
 admin_username = neutron
-admin_password = {{salt['pillar.get']('virl:password', salt['grains.get']('password', 'password')) }}
+admin_password = {{ virl.ospassword }}
 admin_auth_url = http://{{ virl.controller_ip }}:35357/{{ virl.keystone_auth_version }}
 {% endif %}
 
+[vnc]
+vncserver_listen = {{ virl.int_ip }}
```

Comment:
Btw - keeping these two values on int_ip is what the actual deal of the original VNC issue was.